ui/apps/platform/src/Components/EmailTemplate/EmailTemplateForm.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { ReactElement } from 'react';
+import {
+    Button,
+    Flex,
+    FlexItem,
+    Form,
+    FormGroup,
+    FormHelperText,
+    HelperText,
+    HelperTextItem,
+    Text,
+    TextArea,
+    TextContent,
+    TextInput,
+} from '@patternfly/react-core';
+import { FormikContextType, FormikErrors, FormikTouched } from 'formik';
+import get from 'lodash/get';
+
+import {
+    EmailTemplateFormData,
+    maxCustomBodyLength,
+    maxCustomSubjectLength,
+} from './EmailTemplate.utils';
+
+function getFieldValidated(
+    errors: FormikErrors<EmailTemplateFormData>,
+    touched: FormikTouched<EmailTemplateFormData>,
+    fieldId: string
+) {
+    const isFieldInvalid = !!(get(errors, fieldId) && get(touched, fieldId));
+    const fieldValidated = isFieldInvalid ? 'error' : 'default';
+    return fieldValidated;
+}
+
+export type EmailTemplateFormProps = {
+    customBodyDefault: string;
+    customSubjectDefault: string;
+    formik: FormikContextType<EmailTemplateFormData>;
+};
+
+function EmailTemplateForm({
+    customBodyDefault,
+    customSubjectDefault,
+    formik,
+}: EmailTemplateFormProps): ReactElement {
+    const {
+        errors,
+        handleBlur,
+        handleChange,
+        handleSubmit,
+        isSubmitting,
+        setFieldValue,
+        touched,
+        values,
+    } = formik;
+
+    const isSubjectValidated = getFieldValidated(errors, touched, 'customSubject');
+    const isBodyValidated = getFieldValidated(errors, touched, 'customBody');
+
+    return (
+        <Form className="pf-v5-u-py-lg pf-v5-u-px-lg" onSubmit={handleSubmit}>
+            <FormGroup label="Email subject" fieldId="customSubject">
+                <TextInput
+                    id="customSubject"
+                    type="text"
+                    value={values.customSubject}
+                    validated={isSubjectValidated}
+                    onChange={(e) => handleChange(e)}
+                    onBlur={handleBlur}
+                    isDisabled={isSubmitting}
+                    placeholder={customSubjectDefault}
+                />
+                <FormHelperText>
+                    <HelperText>
+                        <HelperTextItem variant={isSubjectValidated}>
+                            {errors.customSubject}
+                        </HelperTextItem>
+                    </HelperText>
+                </FormHelperText>
+                <Flex>
+                    <FlexItem flex={{ default: 'flex_1' }}>
```

Comment:
Not a change request just wanted to point out that this patternfly prop always confuses me. I feel like using either `flex-grow` here of `align-self: end` on the button feel more intuitive to me. Or even `justify-content: space-between` but that could cause more issues if another element is added.